src/main/java/com/example/course/controller/validatorSubject.java

```diff
@@ -0,0 +1,67 @@
+package com.example.course.controller;
+
+
+public class validatorSubject {
+    static Boolean isSubject(String subject)
+    {
+        Boolean val=subject!=null;
+
+        for(int letter=0 ; letter<subject.length() && val ; letter++)
+            if(!( subject.charAt(letter) == ' ' || Character.isAlphabetic(subject.charAt(letter)) ))
```

Comment:
break the loop or return false, unnecessary to iterate over after getting the result